internal/nomad/nomad.go

```diff
@@ -318,9 +318,6 @@ func handleEvaluationEvent(evaluations map[string]chan error, event *nomadApi.Ev
 // storage the state is persisted between multiple calls of this function.
 func handleAllocationEvent(startTime int64, allocations storage.Storage[*allocationData],
 	event *nomadApi.Event, callbacks *AllocationProcessoring) error {
-	if event.Type != structs.TypeAllocationUpdated {
```

Comment:
Would it make sense to set another filter for the `PlanResult` here, rather than removing it?

Reply:
So far in all Influx events, `PlanResult` and `AllocationUpdated` are the only types of events within the `Allocation` topic.

But yeah, let's add the condition with the two topics, and create a Sentry warning in case of a new noticed type?!

Reply:
I was rather thinking of _just_ filtering for `PlanResult`. This would also resolve the issue of duplicated log statements (see the other comment). However, if you say that both events are useful, I would rather follow your suggestion.

Reply:
Ah, I see. At this point, we cannot filter for just `PlanResult` as we depend on the event `Allocation`, `AllocationUpdated`, `running` (ClientStatus).

But, we could in line 361 (see the other comment). I don't know if that would be useful because with our current condition we are deciding on the state of our data, otherwise we would decide on the believe that just the `pending` `AllocationUpdated` event should be filtered out..

Reply:
Mh, if you are saying "we depend on the event Allocation, AllocationUpdated, running (ClientStatus)", how this it work then? This somewhat implies that listening to the `PlanResult` does not provide all information?

Reply:
Right. Our process is split depending on the Client Status

https://github.com/openHPI/poseidon/blob/f377b1376c59a4f3f2a95258678e917e9fef69c5/internal/nomad/nomad.go#L343-L347

We need to handle the `pending` case in order to measure the startup time, and to handle container re-allocations.
We need to handle the `running` case in order to add just ready runners to the idle runners.

For both Client status, we observed 1 to 4 events [[1]](https://github.com/openHPI/poseidon/files/11462924/different_number_of_nomad_events.log). Therefore, we need to filter for just the first event each.

We cannot use only one type (`AllocationUpdated`, `PlanResult`) as we have many `pending` allocations with just the `PlanResult` event, and a couple of `pending` allocations with just the `AllocationUpdated` event [[2]](https://github.com/openHPI/poseidon/files/11463115/one_type_of_pending_events.txt).

Reply:
Okay, interesting. This somewhat sounds like we need to combine both events, in order to get a complete picture? Probably, this supports your proposal to switch to the `Job` events rather than the `Allocation` events.

Let's give it a try and see what happens :)